bthome_v1.html

```diff
@@ -1175,11 +1150,12 @@ <h4>Events</h4>
                 <td>
                   <code>0x02</code>
                 </td>
-                <td>turn_off</td>
+                <td>single_press</td>
+                <td></td>
                 <td>
-                  <code>023B02</code>
+                  <code>023A02</code>
                 </td>
-                <td>Switch event "turn_off"</td>
+                <td>single_press</td>
```

Comment:
why do we have `press` and `single_press`

Reply:
Yeah, that does not really make a lot sense. Removed `single_press`. 
Also `long_single_press` is removed, as we have `long_press`